### DISCUSSION TITLE
I need help wit my Sh1mmer not opening any sh1mmer utilities like unenrolling

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#601] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I js get a failed message whenever i try to unenroll how do it fix

**Comment by cooltoo37** [comment]
Bro First What is your channel I think maybe you have done it wrong also just retry the whole process Ik its not easy I did mine last night whit my stupid slow usb it took me 2 hours  so if it dosent fix then its one of the other wrong channel or pathced also have you built it.

**Reply by ghost** [reply]
i need to downgrade my laptop then ill do it because its patched on 111 rn

**Reply by NightScripts-API** [reply]
You need to downgrade to version 109.0.0 - 83.4.1 

You can use the same type of way you created your SH1M and use it to downgrade then after downgrading just use the Anti-Version upgrader
